Tvl.VisualStudio.MouseFastScroll/source.extension.vsixmanifest

```diff
@@ -14,13 +14,26 @@
     <Tags>mouse, scroll, zoom</Tags>
   </Metadata>
   <Installation>
-    <InstallationTarget Version="[11.0,18.0)" Id="Microsoft.VisualStudio.Community" />
-    <InstallationTarget Version="[11.0,18.0)" Id="Microsoft.VisualStudio.Pro" />
+    <InstallationTarget Version="[11.0,17.0)" Id="Microsoft.VisualStudio.Community" />
+    <InstallationTarget Version="[11.0,17.0)" Id="Microsoft.VisualStudio.Pro" />
+    <InstallationTarget Version="[17.0,18.0)" Id="Microsoft.VisualStudio.Community">
+      <ProductArchitecture>x86</ProductArchitecture>
+    </InstallationTarget>
+    <InstallationTarget Version="[17.0,18.0)" Id="Microsoft.VisualStudio.Community">
+      <ProductArchitecture>amd64</ProductArchitecture>
+    </InstallationTarget>
+    <InstallationTarget Version="[17.0,18.0)" Id="Microsoft.VisualStudio.Pro">
+      <ProductArchitecture>x86</ProductArchitecture>
+    </InstallationTarget>
+    <InstallationTarget Version="[17.0,18.0)" Id="Microsoft.VisualStudio.Pro">
+      <ProductArchitecture>amd64</ProductArchitecture>
+    </InstallationTarget>
```

Comment:
Why do you need Pro here? Targeting Community allow it to work on higher SKUs.

Reply:
I'm guessing I don't need it separated for 17+. There were some earlier versions where Community wasn't recognized.

Reply:
16.0 didn't need it either AFAIK. I think we only needed special handling like this back when we had the Express SKUs.

Reply:
This extension (like many of my extensions) supports Visual Studio 2012 and newer.